CorsixTH/Lua/movie_player.lua

```diff
@@ -233,8 +233,8 @@ function MoviePlayer:playMovie(filename, wait_for_stop, can_skip, callback)
   warning = self.moviePlayer:play(self.channel)
   if warning ~= nil and warning ~= "" then
     local message = "MoviePlayer:playMovie - Warning: " .. warning
-    if self.app.world then
-      self.app.world:gameLog(message)
+    if self.app then
```

Comment:
Can you not have an `app` while having a `moviePlayer` ?
I can see that happening for `world` (intro movie), but `app` feels much more questionable to me.